clang/lib/StaticAnalyzer/Checkers/CXXDeleteChecker.cpp

```diff
@@ -220,11 +220,11 @@ CXXDeleteChecker::PtrCastVisitor::VisitNode(const ExplodedNode *N,
                                                     /*addPosRange=*/true);
 }
 
-void ento::registerCXXArrayDeleteChecker(CheckerManager &mgr) {
+void ento::registerArrayDeleteChecker(CheckerManager &mgr) {
```

Comment:
Why was the `CXX` removed here if the checker is still named `cplusplus.ArrayDelete`.

Reply:
Brings it more in line with similar C++ checkers, see eg. `DeleteWithNonVirtualDtorChecker` in the same file.

The only place where this name matters is the `Checkers.td` where it is under the `cplusplus` group anyways.